content/ru/docs/reference/kubectl/docker-cli-to-kubectl.md

```diff
@@ -0,0 +1,365 @@
+---
+title: kubectl для пользователей Docker
+content_template: templates/concept
+reviewers:
+- brendandburns
+- thockin
```

Comment:
When localizing, remove reviewers unless you know they speak the target language well enough to review.